src/osal/lv_freertos.c

```diff
@@ -15,11 +15,7 @@
 #include "lv_os.h"
 #if LV_USE_OS == LV_OS_FREERTOS
 
-#if (ESP_PLATFORM)
-    #include "freertos/atomic.h"
-#else
-    #include "atomic.h"
-#endif
+#include "atomic.h"
```

Comment:
With ESP IDF I get `fatal error: atomic.h: No such file or directory`

Reply:
I removed that because the file name is the same, only the path is different.
The suggestion how to fix it is in the commit message, can you please try it out?
[fix(freertos) Use the proper include path("C_INCLUDE_PATH" or "GCC -I") instead.](https://github.com/lvgl/lvgl/pull/6793/commits/3237397062e163148b7700119890a129bea06ab3)

Reply:
I got it to work by adding

```cmake
if(CONFIG_FREERTOS_SMP)
    target_include_directories(${COMPONENT_LIB} PRIVATE "${IDF_PATH}/components/freertos/FreeRTOS-Kernel-SMP/include/freertos/")
else()
    target_include_directories(${COMPONENT_LIB} PRIVATE "${IDF_PATH}/components/freertos/FreeRTOS-Kernel/include/freertos/")
endif()
```

to the bottom of `env_support/cmake/esp.cmake`. Please add it.

Reply:
Done: [fb975da](https://github.com/lvgl/lvgl/pull/6793/commits/fb975daae68d0e1699bf03226f99a465d0b3614f)